Resources/Prototypes/tags.yml

```diff
@@ -245,6 +245,9 @@
 
 - type: Tag
   id: Plastic
+  
+- type: Tag
+  id: Pickaxe
```

Comment:
Alphabetical please.

Reply:
Plastic was out of order, I put it in the proper spot, so it's now in proper alphabetical order.